### ISSUE TITLE
ANH Ibiza hub - Test Send Payload Accessibility Issues

### ISSUE THREAD

**Issue by Amit8527510735** [microsoft/monaco-editor#1519]
Title: WCAG 1.4.3: Elements must have sufficient color contrast (.mtk3) Tags: Accessibility, WCAG 1.4.3, color-contrast Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI) Target application title: Test Send - Microsoft Azure Target application url: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/dd2892d2-5741-4fcb-a898-e78f8df68c0e/resourceGroups/JMW-NH/providers/Microsoft.NotificationHubs/namespaces/JMWNS01/notificationHubs/JMW-Test/testSend Element path: .mtk3 Snippet: <span class="mtk3">xml</span> How to fix: Fix any of the following:   Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #fffffe, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1 Environment: Chrome version 73.0.3683.86 ==== This accessibility issue was found using Accessibility Insights for Web 2.0.5 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Title: WCAG 1.4.3: Elements must have sufficient color contrast (.mtk4:nth-child(4)) Tags: Accessibility, WCAG 1.4.3, color-contrast Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI) Target application title: Test Send - Microsoft Azure Target application url: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/dd2892d2-5741-4fcb-a898-e78f8df68c0e/resourceGroups/JMW-NH/providers/Microsoft.NotificationHubs/namespaces/JMWNS01/notificationHubs/JMW-Test/testSend Element path: .mtk4:nth-child(4) Snippet: <span class="mtk4">version</span> How to fix: Fix any of the following:   Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #fffffe, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1 Environment: Chrome version 73.0.3683.86 ==== This accessibility issue was found using Accessibility Insights for Web 2.0.5 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Title: WCAG 1.4.3: Elements must have sufficient color contrast (.mtk4:nth-child(8)) Tags: Accessibility, WCAG 1.4.3, color-contrast Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI) Target application title: Test Send - Microsoft Azure Target application url: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/dd2892d2-5741-4fcb-a898-e78f8df68c0e/resourceGroups/JMW-NH/providers/Microsoft.NotificationHubs/namespaces/JMWNS01/notificationHubs/JMW-Test/testSend Element path: .mtk4:nth-child(8) Snippet: <span class="mtk4">encoding</span> How to fix: Fix any of the following:   Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #fffffe, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1 Environment: Chrome version 73.0.3683.86 ==== This accessibility issue was found using Accessibility Insights for Web 2.0.5 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Re-pro steps:
1. Open https://ms.portal.azure.com and login via valid credentials. 
2. Navigate to More Services and select it.                                 
3. Search "Notification hub".
4. Navigate to  any "Notification Hub" in Name column of Notification Hubs pane and select it.   
5. Select  ‘Overview’ from Menu and Select Test send from the command bar on top.                                                                                                                                  
6. Test Send pane opens, navigate on Platforms edit box and select Windows Phone/Windows.
7. Navigate on the controls of test send pane and enter the details in the edit fields.
8. Navigate to 'send' control and select it. Sending notifications will appear on the top right of the page.

![image](https://user-images.githubusercontent.com/53263029/61793678-42e43180-ae3d-11e9-8a72-c05de4d92fa5.png)

**Comment by alexandrudima** [comment]
Please open the issue against the Azure Portal directly. The editor comes with a high contrast theme that the Azure Portal can employ to get high contrast.
